### ISSUE TITLE
newer:potomo

### ISSUE THREAD

**Issue by stebrech** [WPZOO/grunt#2]
Task is not working for me. No error message is showing.

```
Running "newer:potomo" (newer) task
Running "newer:potomo:dist" (newer) task
No newer files to process.
```

**Comment by grappler** [comment]
It should be fixed with https://github.com/WPZOO/grunt/commit/0fbdc01e3ebeeccae52f10503f797e38735862d8 Let me know if it helps

**Comment by stebrech** [comment]
Fixed with df20995
